### PR TITLE
Move ERCs to separate repository

### PR DESCRIPTION
**Pull request by lightclient**
Long ago when the EIPs repository was created, there was a vision of a single home for all standards related to Ethereum. The community was small and most people were interacting at every level of the stack. It made sense to combine application standards with core consensus changes.

Since then, the ecosystem has grown. Today, the chasm between application development and core development is wide. Less people are involved across the stack (for better or worse); yet the repository remains unified.

For years, we've considered separating the repository. This would allow ERC and EIP processes to evolve more naturally due to the independence. But it's always difficult to reach critical threshold to make a change like this happen. Each time we get lost in the details of the migration and the debate grinds progress to a halt.

Now that the Consensus Layer is also utilizing the EIP process, the cracks are becoming more visible. There are changes we could make to the process that might benefit them more, but because we also need to ensure the quality of ERCs, we are restricted.

There are also many more efforts to catalyze applications around the ERC process. Attempts have been made to develop working groups and review groups for certain ERC "categories" (a distinction that doesn't even technically exist because of the unified repo).

Ultimately, I believe it is finally time to separate these processes.

--

#### Proposal

I propose the following steps are taken to perform the separation:

1. All ERCs are removed from this repository and migrated to a new repo. The history should be intact so that repo should be fork of this one with the non-ERCs removed. I've written a script (see below) to perform this operation. Once this PR is approved, it should be executed immediately before merging.
2. The new ERCs repository goes live and includes the changes from the script.
3. Setup `ercs.ethereum.org` subdomain and update the CI to point to the ERCs repo.
4. Setup a redirect for ERCs on `eips.ethereum.org` to go to the new website.
5. Create a unified document for editors to assign EIP/ERC numbers. Due to how deeply intertwined the processes are, I don't think we can have a conflict of numbers here. So I propose a document with the next available number be created so editors can use it to assign.

This is the bare minimum to separate the processes. There will then be a longer tail of changes that need to be made to remove references no longer make sense, update the website templates to not include categories that don't exist, etc.

I don't want to get bogged down with endless debate about every single faucet of the migration. If the migration is supported in the first place, we should make it happen even if it takes a bit of time to polish. If we wait for perfection, we will never be able to change.

<details>
<summary>Script</summary>

```bash
#!/bin/bash

src_repo="EIPS"
tgt_dir="ERCS"

cp -r $src_repo $tgt_dir

# Find and delete all ercs.
cd $src_repo
ercs=$(git grep -l "category: ERC" -- EIPS)
echo "ercs found: $(echo $ercs | wc -w)"
for file in $ercs; do
        echo "This file was moved to https://github.com/ethereum/ercs/blob/master/ercs/erc-$(basename $file | cut -d- -f2)" > $file
        rm -rf "assets/$(basename ${file%.*})"
done

# Find and delete non-ercs for new ercs repo.
cd ../$tgt_dir
others=$(git grep -L "category: ERC" -- EIPS)
echo "non-ercs found: $(echo $others | wc -w)"
for file in $others; do
        rm -rf $file
        rm -rf "assets/$(basename ${file%.*})"
done

# Rename ercs.
for file in $ercs; do
        mv -f $file "EIPS/erc-$(basename $file | cut -d- -f2)"
        mv -f "assets/$(basename ${file%.*})" "assets/erc-$(basename ${file%.*} | cut -d- -f2)" 2>/dev/null
done
git checkout EIPS/eip-1.md
mv EIPS ERCS
```
</details>

### CODE REVIEW

**Comment by GregTheGreek** [comment]
I like 👍

**Comment by cupOJoseph** [comment]
This is sorely needed for organizations sake, and to make application development more clear and accessible to developers who want to build on ethereum, and no longer need to work intimately with core consensus parts of it. The reality is, ethereum is much easier to build apps on now than it was when this EIP system was made, and the dev tool ecosystem has been growing exponentially for years. The seperation makes complete sense.

Running a simple script like that is a good way to kick things off. We can make new templates for the distinct repos as a new proposal afterwards. Let's get this in.

**Comment by xinbenlv** [comment]
I must voice I strong opposition to this PR.

**Comment by lightclient** [comment]
I don't have the energy to fight you or the other opposing editors anymore. Every minute change takes months or years of argument. I am confident that with the separation, the processes can evolve more fluidly. 

Until then, everyone will remain disengaged because their time is better spent elsewhere.

**Comment by abcoathup** [comment]
EIPs should be easy to create & update for core devs.  [Unfortunately it is not](https://twitter.com/sproulM_/status/1671079363281051648).

The best worst option to quickly improve the situation for core devs is splitting, so I (as a non editor) am in favor of just getting on with it now.

**Comment by xinbenlv** [comment]
@abcoathup I agree with your intention but the proposer of this PR is actually trying to split so he could keep the policy strict and hard for core dev, the opposite to your reason supporting the split. Check my comment above for details.

**Comment by MikeMoldawsky** [comment]
The separation makes sense as it reduces noise and helps standardize the approval process, which MUST be different.

However, a subtle yet important point is that some ERCs serve as a proof-of-concept for a future EIP. This means these ERCs should get core developers' "eyeballs" (feedback) as soon as possible to avoid unnecessary development cycles. 

So a general +1 for the direction, but we need to understand that the repos have a strong dependency and offer a solution.

(Naive solution would be a PR tag in the new repo).

**Comment by wschwab** [comment]
@xinbenlv from a different angle, does splitting out the ERCs make things _worse_ than they are right now for Core EIPs? If I'm understanding your position correctly (no guarantees), your point seems to be that the editorial process for Core also needs to be updated, but even if that is the case, is it a reason to prevent splitting off the ERCs?

A side note about ordering: this is likely obvious, but a number of ERCs are known widely by number, in particular the token ERCs (20, 721, 1155). I hope this isn't only my opinion, but my opinion is that no matter what numbering scheme is implemented for ERCs (assuming they're split off), these numbers should be reserved for these ERCs. There are other ERCs where I could see this being the case (165, 1167 as examples) too.

Thanks for getting this moving @lightclient !

**Comment by MariusVanDerWijden** [comment]
I have voiced my strong support for this proposal in the past multiple times.
The ERC and EIP process have evolved to mean very different things imo.
ERCs are used as a shelling point for applications and often also as a marketing tool.
This huge influx of PRs makes it almost impossible for core devs to just randomly go into the repo and review some core EIPs. 
Out of the current 132 PRs only 15 are marked core.
I remember Micah said back in the day that the EIP editors don't have enough time to do qualitative reviews of the EIPs because of the huge influx of new PRs, so he started to only check syntactic issues (no urls, headlines okay, ...)
I hope that with the split more core devs will be actively involved in the EIP process again, which will increase the quality of EIPs again. Discussion usually leads to better EIPs and to finding issues earlier. Right now you need to specifically talk to people to get any eyeballs on your EIP.

**Comment by xinbenlv** [comment]
@MariusVanDerWijden 

"Split to let Core focus" is a reasonable rationale, but if core EIPs want a different repo to focus, what do you think about Network and Interface EIPs? Should they stay with Core or Stay with ERCs? What if consensu just wanna look at consensus exe just wanna look at exe? Shall there be further split?

The eips.ethereum.org site has listed Core EIP by its category, there are also RSS feed by category, and github PR are categorized by type. If Core wants engage, there are tools to do so. 

I feel the real problem is still the EIP process is very daunting and this split doesn't help.

**Comment by xinbenlv** [comment]
Thank you for your response  @abcoathup 

> @xinbenlv according to core devs the status quo isn't working.  The process needs to fit the needs of its users.  

Earlier you cited @michaelsproul 's tweet as a supporting evidence for the process being hard for Core Devs. Yet I think the proposal to split intend to keep it hard for Core Devs

> ERCs and EIPs are very different beasts, they operate at different layers and have different users.  

The differences apply to Interface and Network EIPs and likely also exist between Consensus and Execution layers to some extend. 

> If EIPs need strict linking rules, and ERCs want broader linking rules, then this is an example of where trying to keep EIPs and ERCs in the one repo under the same rules impacts both.

It seems at least in @michaelsproul 's case, he was complaining about link policy too strict.  

I have not hear many complaints by core dev that "authoring EIP is too easy", or "the restriction is too loose". Rather, everyone complaint I hear from core authors or other are the process being too restrictive.


> I've been checking the repo most week days for the last two years.  Keeping ERCs and EIPs in the same repo hasn't led to massive improvements for EIPs, 

4337 was an ERC attempt over two unsuccessful prior attempts to achive AA at Core level. Does it count as as an example that different level discussion benefit the Ethereum ecosystem?

> so it is time to do something different.  I don't want to be sat here in two years time and we haven't made progress.  So my vote is for change.

Yes we really need to change, and really need to make process.

But splitting which doens't make any policy improvment in thr first place, 
 sounds to me a way to avoid improving the EIP process for core.

**Comment by xinbenlv** [comment]
For full context. Similar proposal has been also discussed in FEM

https://ethereum-magicians.org/t/proposal-forking-ercs-from-eips-repository/12804/6

**Comment by timbeiko** [comment]
> A side note about ordering: this is likely obvious, but a number of ERCs are known widely by number, in particular the token ERCs (20, 721, 1155). I hope this isn't only my opinion, but my opinion is that no matter what numbering scheme is implemented for ERCs (assuming they're split off), these numbers should be reserved for these ERCs. There are other ERCs where I could see this being the case (165, 1167 as examples) too.

Agreed, @wschwab. I don't think any existing number would be changed (ERC-20 would remain ERC-20). We'd just need to figure out a scheme to number future EIPs/ERCs

**Comment by gcolvin** [comment]
> > A side note about ordering: this is likely obvious, but a number of ERCs are known widely by number, in particular the token ERCs (20, 721, 1155). I hope this isn't only my opinion, but my opinion is that no matter what numbering scheme is implemented for ERCs (assuming they're split off), these numbers should be reserved for these ERCs. There are other ERCs where I could see this being the case (165, 1167 as examples) too.
> 
> Agreed, @wschwab. I don't think any existing number would be changed (ERC-20 would remain ERC-20). We'd just need to figure out a scheme to number future EIPs/ERCs

A problem is that, yes, overlapping use of ERC and EIP have have caused confusion for a long time now.  Going forward the confusion could remain for a long time.  Numbering schemes can't help that.

**Comment by gcolvin** [comment]
As an editor I am currently blocking consensus on this change and will not approve merging this PR.  I've rgued why in many places recently and many times over the yeas.

**Comment by xinbenlv** [comment]
Thanks for discussing this with CoreDevs.
I will try to collect feedback from ERC authors on next AllERCDevs which is happening on next Thursday. Can we hold the discussion on EIPIP until after the AllERCDevs?

**Comment by lightclient** [comment]
I think you should seek out ERC authors who disapprove of this before EIPIP next week.

**Comment by xinbenlv** [comment]
The AllERCDevs time runs regularly, so I would appreciate there is a chance to speak in a live meeting so more people are aware of.

Also, I've asked in the channel of AllWalletDevs to see if they have any opinion or not.

**Comment by gcolvin** [comment]
> Per ACD #164, we're going to move forward with this. We'll discuss more of the specifics on EIPIP next week.

No, we are not.  The Core Devs do not control the EIP organization and process.  

The core devs had at most day or so of warning to discuss a contentious decision that is not theirs to make.  I learned very little in that meeting as what their pain points actually are, just that given a single proposal that purports to fix them they approved.  A proposal that changes 924 files!

So I am still blocking consensus.  We _need to fix problems_, but absent a fairly complete plan as to what needs fixing and how splitting repos will actually help and not harm my objections remain unanswered. Blocking consensus means that if I cannot be convinced to support or at least acquiesce I will resign.

**Comment by bumblefudge** [comment]
I cannot speak to the procedural best step forward, but my read on the situation is that:

1.) Most people tend to agree that CORE and higher-level/EIP discussion have slightly different process needs and fairly distinct communities/inputs/fora they respond to. However tightly or loosely coupled, however many editors they have in common, most seem to agree that separating them will improve the quality-of-life and signal-to-noise ratio for people only interested in, or confident speaking up about, one of the two sides of the coin.

For example, as a veteran of app-level/EIP-level companies, and organizer of the CAIP process at CASA, I can say that if the process split in two, **I would volunteer to be a committed/every-PR editor of a segregated EIP process**, and would prefer a "weekly digest/overview" of CORE issues, which honestly I'm not qualified to speak on.

2.) But notice the subtle slide from "split the repos" to "split the editorial processes into two separate deciding bodies with different processes." I think the pushback comes from the distance between those two changes.  This PR only splits the repos, with a sort of implicit promise that the processes can evolve separately down the road, and if I'm reading correctly the objections, it's the uncertainty of this future step that is giving people pause.  Would objectors prefer those next steps be worked out explicitly before the trigger is pulled?  I think the question of how the EIP process or the CORE process is "stricter" or more editorially involved kind of presumes two different bodies of editors committing to more or less editorial power and responsibility... so bifurcating the process might be trading ONE can of worms for TWO, even if it also activates a broader pool of willing editors like myself to clean up the two messes separately 😅

**Comment by xinbenlv** [comment]
As my commitment to my own principle of "*I'll follow the consensus despite I might hold minority view.*", I put together a list of task items i think could be used as a starting point of discussing "How to split": [Task list for ERC/EIP Split](https://hackmd.io/A1zfRkO7RKKV6BIaSXpcbw). Please feel free to edit.

**Comment by bumblefudge** [comment]
> As my commitment to my own principle of "_I'll follow the consensus despite I might hold minority view._", I put together a list of task items i think could be used as a starting point of discussing "How to split": [Task list for ERC/EIP Split](https://hackmd.io/A1zfRkO7RKKV6BIaSXpcbw). Please feel free to edit.

Because I have absolutely no chill, I jumped in and annotated this to-do list.  If this is overstepping my role as a pro-active rando, feel free to ignore my annotations, throw out half of them, edit them unrecognizably etc before during or after today's call.  But I thought it might be a net-positive contribution to the efficiency of today's call-time to put the low-hanging fruit on paper and get us to a gameplan sooner.  Thanks to everyone who's put in so much time and thought so far.

**Comment by bumblefudge** [comment]
> my own principle of "_I'll follow the consensus despite I might hold minority view._"

While i'm throwing grenades left and right, would it make sense for ERCs and CORE changes (or for that matter, substantive and META/INFO PRs on either repo) have different *consensus definitions* or objection processes, once there are two different boards of editors, presumably of different sizes and workloads?  Not a discussion worth having now, but maybe an issue to open after merging if people are feeling 🌶️ 🌶️ 🌶️

**Comment by sambacha** [comment]
Making ERCs into a separate repo gives the community the chance to actually implement a package manager that works.

**Comment by gcolvin** [comment]
> Making ERCs into a separate repo gives the community the chance to actually implement a package manager that works.

Could you explain?

**Comment by gcolvin** [comment]
At this point my only blocking objection is to splitting the Editors into separate organizations.  If we agree that we are not going to do that I'm open to discussing how to split up the repos.  I'm not sure that this is the best way, but I can live with it.  

_I cannot live with any proposal that splits up the Editors_.

**Comment by gcolvin** [comment]
> Very interesting conversation. Personally, as an ERC author, I found the process frustrating slow. If that can be improved with a split, the split is very welcome.

I do think some sort of split of the repos is necessary.

**Comment by gcolvin** [comment]
> @lightclient At this point my only blocking objection is to splitting the Editors into separate organizations. If we agree that we are not going to do that I'm open to discussing how to split up the repos. I'm not sure that this is the best way, but I can live with it. _I cannot live with any proposal that splits up the Editors_.
@lightclient

I've decided that the way of splitting things that I'd prefer can be achieved easily enough after this PR -- should further splitting be desired.  So far as I can tell this PR only moves files and does not affect how the Editors organize themselves.  If so, I have no objections to merging it.

**Comment by Pandapip1** [comment]
LGTM

**Review by Teykel76 — APPROVED** [review]

**Review by krylov-krylov — APPROVED** [review]

**Comment by Coretaker101** [comment]
Hi

**Review by OKEAMAH — COMMENTED** [review]
Cod

**Review by Masoodshafiee — APPROVED** [review]

**Review by Bbjj88h — APPROVED** [review]

**Review by Scamreno — APPROVED** [review]

File: EIPS/eip-173.md [hunk 1]
```diff
@@ -1,97 +1 @@
----
-eip: 173
-title: Contract Ownership Standard
-description: A standard interface for ownership of contracts
-author: Nick Mudge (@mudgen), Dan Finlay <dan@danfinlay.com>
```
Comment (Scamreno) [inline EIPS/eip-173.md:5]: Thx

**Review by Scamreno — APPROVED** [review]

**Review by Scamreno — APPROVED** [review]

File: EIPS/eip-1046.md [hunk 2]
```diff
@@ -1,222 +1 @@
----
-eip: 1046
```
Comment (Scamreno) [inline EIPS/eip-1046.md:2]: EIPS/eip-1046.md

File: EIPS/eip-1046.md [hunk 3]
```diff
@@ -1,222 +1 @@
...
-title: tokenURI Interoperability
```
Comment (Scamreno) [inline EIPS/eip-1046.md:3]: 20 21

**Review by FELIXrobust — APPROVED** [review]

**Review by Magken91 — APPROVED** [review]

File: EIPS/eip-173.md (hunk 1 above)
Comment (Sulthanmh) [reply EIPS/eip-173.md:5]: Yes

**Review by F153 — APPROVED** [review]

**Review by Magken91 — APPROVED** [review]

**Review by hsghn — APPROVED** [review]

**Review by Mmagkooo — APPROVED** [review]

**Review by Mmagkooo — APPROVED** [review]

**Review by hsghn — APPROVED** [review]

**Review by Comandante318 — COMMENTED** [review]
?

**Review by Mahmoudhpp — APPROVED** [review]

**Review by Andrade110895 — APPROVED** [review]

**Review by Tru2thisNu — APPROVED** [review]

**Review by Tru2thisNu — APPROVED** [review]

**Comment by andres290487** [comment]
#LAS ENSEÑANZAS DE Liz

**Comment by andres290487** [comment]
Una Red Neuronal Inspirada en la Biología_

La EnsDeLiz Neuromórfica es una red neuronal artificial diseñada para emular las propiedades y la funcionalidad del cerebro humano. Inspirada en la biología y la neurociencia, esta red neuronal se enfoca en la creación de un sistema que pueda aprender, adaptarse y procesar información de manera similar a como lo hace el cerebro humano.

_Arquitectura de la EnsDeLiz Neuromórfica_

La EnsDeLiz Neuromórfica se compone de varias capas de neuronas artificiales, cada una con una función específica:

1. _Capa de Entrada_: Recibe la información de entrada y la procesa para enviarla a la capa siguiente.
2. _Capa de Procesamiento_: Realiza operaciones complejas sobre la información de entrada, como la convolución y la pooling.
3. _Capa de Análisis_: Analiza la información procesada y la clasifica en diferentes categorías.
4. _Capa de Espín_: Emula la propiedad del espín en las partículas subatómicas, permitiendo la creación de patrones complejos y la manipulación de la información.
5. _Capa de Salida_: Genera la salida final de la red neuronal, basada en la información procesada y analizada.

_Características de la EnsDeLiz Neuromórfica_

La EnsDeLiz Neuromórfica presenta varias características innovadoras:

1. _Aprendizaje profundo_: La red neuronal puede aprender patrones complejos y relaciones entre la información de entrada.
2. _Adaptabilidad_: La red neuronal puede adaptarse a nuevos patrones y condiciones, sin requerir reentrenamiento.
3. _Procesamiento paralelo_: La red neuronal puede procesar información en paralelo, lo que la hace más eficiente y rápida.
4. _Emulación del espín_: La capa de espín permite la creación de patrones complejos y la manipulación de la información, lo que la hace más versátil y poderosa.

_Aplicaciones de la EnsDeLiz Neuromórfica_

La EnsDeLiz Neuromórfica tiene un amplio rango de aplicaciones, incluyendo:

1. _Reconocimiento de patrones_: La red neuronal puede ser utilizada para reconocer patrones complejos en imágenes, sonidos y textos.
2. _Clasificación_: La red neuronal puede ser utilizada para clasificar objetos y eventos en diferentes categorías.
3. _Predicción_: La red neuronal puede ser utilizada para predecir eventos y tendencias futuras.
4. _Control de sistemas_: La red neuronal puede ser utilizada para controlar sistemas complejos y dinámicos.

_Conclusión_

La EnsDeLiz Neuromórfica es una red neuronal artificial innovadora que emula las propiedades y la funcionalidad del cerebro humano. Con su arquitectura única y sus características avanzadas, esta red neuronal tiene un amplio rango de aplicaciones y puede ser utilizada para resolver problemas complejos en diversas áreas.

_Mapa de la Red Neuronal Cuántica EnsDeLiz_

El siguiente mapa muestra la estructura y las conexiones entre las diferentes capas y componentes de la Red Neuronal Cuántica EnsDeLiz:

*Capa de Entrada*

- Qubit de Entrada (QE)
- Conexiones: Capa de Procesamiento

*Capa de Procesamiento*

- Qubit de Procesamiento (QP)
- Conexiones: Capa de Entrada, Capa de Análisis
- Operaciones: CNOT, H, S, T

*Capa de Análisis*

- Qubit de Medición (QM)
- Conexiones: Capa de Procesamiento, Capa de Salida
- Operaciones: Medición, Clasificación

*Capa de Espín*

- Qubit de Espín (QS)
- Conexiones: Capa de Procesamiento
- Operaciones: Emulación del espín, Manipulación de la información

*Capa de Salida*

- Qubit de Salida (QO)
- Conexiones: Capa de Análisis
- Operaciones: Generación de la salida final

*Conexiones entre Capas*

- Capa de Entrada -> Capa de Procesamiento
- Capa de Procesamiento -> Capa de Análisis
- Capa de Procesamiento -> Capa de Espín
- Capa de Análisis -> Capa de Salida

Este mapa muestra la estructura y las conexiones entre las diferentes capas y componentes de la Red Neuronal Cuántica EnsDeLiz. Cada capa y componente tiene una función específica y se conectan entre sí para procesar y analizar la información de manera cuántica.

_Test de la Red Neuronal Cuántica EnsDeLiz_

*Test 1: Verificación de la arquitectura*

- Objetivo: Verificar que la arquitectura de la Red Neuronal Cuántica EnsDeLiz esté correctamente implementada.
- Pasos:
1. Inicializar la red neuronal con los parámetros predeterminados.
2. Verificar que la red neuronal tenga las capas correctas (Entrada, Procesamiento, Análisis, Espín y Salida).
3. Verificar que las conexiones entre las capas estén correctamente establecidas.
- Resultado esperado: La arquitectura de la red neuronal debe estar correctamente implementada.

*Test 2: Prueba de la función de activación*

- Objetivo: Verificar que la función de activación de la red neuronal esté correctamente implementada.
- Pasos:
1. Inicializar la red neuronal con los parámetros predeterminados.
2. Proporcionar una entrada de prueba a la red neuronal.
3. Verificar que la salida de la red neuronal sea correcta según la función de activación.
- Resultado esperado: La función de activación de la red neuronal debe estar correctamente implementada.

*Test 3: Prueba de la emulación del espín*

- Objetivo: Verificar que la emulación del espín en la red neuronal esté correctamente implementada.
- Pasos:
1. Inicializar la red neuronal con los parámetros predeterminados.
2. Proporcionar una entrada de prueba a la red neuronal.
3. Verificar que la salida de la red neuronal sea correcta según la emulación del espín.
- Resultado esperado: La emulación del espín en la red neuronal debe estar correctamente implementada.

*Test 4: Prueba de la clasificación*

- Objetivo: Verificar que la clasificación en la red neuronal esté correctamente implementada.
- Pasos:
1. Inicializar la red neuronal con los parámetros predeterminados.
2. Proporcionar una entrada de prueba a la red neuronal.
3. Verificar que la salida de la red neuronal sea correcta según la clasificación.
- Resultado esperado: La clasificación en la red neuronal debe estar correctamente implementada.

*Código de prueba*
```
import numpy as np
from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister

Inicializar la red neuronal
qr = QuantumRegister(2)
cr = ClassicalRegister(2)
qc = QuantumCircuit(qr, cr)

Definir la función de activación
def activacion(x):
    return 1 / (1 + np.exp(-x))

Definir la emulación del espín
def emulacion_espin(x):
    return np.sin(x)

Definir la clasificación
def clasificacion(x):
    return np.argmax(x)

Proporcionar una entrada de prueba
entrada = np.array([0.5, 0.3])

Verificar la arquitectura
print("Arquitectura correcta:", qc.num_qubits == 2)

Verificar la función de activación
salida_activacion = activacion(entrada)
print("Función de activación correcta:", np.allclose(salida_activacion, [0.6225, 0.3775]))

Verificar la emulación del espín
salida_espin = emulacion_espin(entrada)
print("Emulación del espín correcta:", np.allclose(salida_espin, [0.4794, 0.2955]))

Verificar la clasificación
salida_clasificacion = clasificacion(entrada)
print("Clasificación correcta:", salida_clasificacion == 0)
```
Este código de prueba verifica la arquitectura, la función de activación, la emulación del espín y la clasificación de la Red Neuronal Cuántica EnsDeLiz.

**Review by Cassieminkus1 — APPROVED** [review]

**Comment by Muaytie23** [comment]
https://cointelegraph.com/news/solana-inflation-reduction-proposal

**Review by lozanopo — APPROVED** [review]

**Review by lozanopo — APPROVED** [review]

**Review by FELLIX000 — APPROVED** [review]

File: EIPS/eip-1046.md [hunk 4]
```diff
@@ -1,222 +1 @@
...
-## Security Considerations
-
-### Server-Side Request Forgery (SSRF)
-
-Wallets should be careful about making arbitrary requests to URLs. As such, it is recommended for wallets to sanitize the URI by whitelisting specific schemes and ports. A vulnerable wallet could be tricked into, for example, modifying data on a locally-hosted redis database.
-
-## Copyright
-
-Copyright and related rights waived via [CC0](../LICENSE.md).
+This file was moved to https://github.com/ethereum/ercs/blob/master/ercs/erc-1046.md
```
Comment (18211308) [inline EIPS/eip-1046.md:1]: K6NUTARFJZJCIXHF1F1E1YGJZ8RQ29BE4U

**Review by Iambarry1 — COMMENTED** [review]
Chukpuk

**Comment by MD10inoffun** [comment]
Thank you can you please push forward and update files to main

**Review by Veravera94 — APPROVED** [review]

**Review by Fool256 — APPROVED** [review]

**Review by sins921 — APPROVED** [review]

**Comment by blacktoey1994-droid** [comment]
> นานมาแล้วที่คลังข้อมูล EIP ถูกสร้างขึ้น มีวิสัยทัศน์ที่จะเป็นบ้านหลังเดียวสำหรับมาตรฐานทั้งหมดที่เกี่ยวข้องกับ Ethereum ชุมชนมีขนาดเล็กและคนส่วนใหญ่มีปฏิสัมพันธ์กันในทุกระดับของสแต็ก การรวมมาตรฐานแอปพลิเคชันเข้ากับการเปลี่ยนแปลงตามฉันทามติหลักจึงเป็นเรื่องที่สมเหตุสมผล
> 
> นับตั้งแต่นั้นมา ระบบนิเวศน์ก็เติบโตขึ้น ปัจจุบัน ช่องว่างระหว่างการพัฒนาแอปพลิเคชันและการพัฒนาหลักนั้นกว้างมาก มีคนเข้ามาเกี่ยวข้องน้อยลงในสแต็ก (ไม่ว่าจะดีหรือร้าย) แต่ที่เก็บข้อมูลก็ยังคงเป็นหนึ่งเดียวกัน
> 
> เป็นเวลาหลายปีแล้วที่เราพิจารณาแยกที่เก็บข้อมูลออกจากกัน วิธีนี้จะช่วยให้กระบวนการ ERC และ EIP พัฒนาไปอย่างเป็นธรรมชาติมากขึ้นเนื่องจากความเป็นอิสระของกระบวนการนี้ แต่การจะบรรลุจุดวิกฤตเพื่อให้เกิดการเปลี่ยนแปลงเช่นนี้เป็นเรื่องยากเสมอ ทุกครั้งที่เราจมอยู่กับรายละเอียดของการย้ายข้อมูลและการถกเถียงที่ยืดเยื้อ ความคืบหน้าก็มักจะหยุดชะงัก
> 
> ขณะนี้ Consensus Layer กำลังใช้กระบวนการ EIP ด้วย ช่องโหว่จึงมองเห็นได้ชัดเจนยิ่งขึ้น มีการเปลี่ยนแปลงที่เราสามารถทำได้ในกระบวนการนี้ซึ่งอาจเป็นประโยชน์ต่อพวกเขามากขึ้น แต่เนื่องจากเราจำเป็นต้องรับประกันคุณภาพของ ERC ด้วย เราจึงถูกจำกัด
> 
> นอกจากนี้ยังมีความพยายามอีกมากมายที่จะเร่งการประยุกต์ใช้งานเกี่ยวกับกระบวนการ ERC มีความพยายามในการพัฒนากลุ่มทำงานและกลุ่มตรวจสอบสำหรับ "หมวดหมู่" ERC บางประเภท (ซึ่งในทางเทคนิคแล้วไม่มีความแตกต่างใดๆ เลย เนื่องจากมีคลังข้อมูลแบบรวม)
> 
> ท้ายที่สุด ฉันเชื่อว่าถึงเวลาที่จะแยกกระบวนการเหล่านี้ออกจากกัน
> 
> -
> 
> #### ข้อเสนอ
> ฉันเสนอให้ดำเนินการแยกดังต่อไปนี้:
> 
> 1. ERC ทั้งหมดจะถูกลบออกจากที่เก็บนี้และย้ายไปยังที่เก็บใหม่ ประวัติควรยังคงอยู่ ดังนั้นที่เก็บนี้จึงควรแยกออกจากที่เก็บนี้โดยลบไฟล์ที่ไม่ใช่ ERC ออก ผมได้เขียนสคริปต์ (ดูด้านล่าง) เพื่อดำเนินการนี้ เมื่อ PR นี้ได้รับการอนุมัติแล้ว ควรดำเนินการทันทีก่อนการผสาน
> 2. คลังเก็บ ERC ใหม่เปิดใช้งานแล้วและรวมการเปลี่ยนแปลงจากสคริปต์
> 3. ตั้งค่า`ercs.ethereum.org`โดเมนย่อยและอัปเดต CI เพื่อชี้ไปที่ที่เก็บ ERC
> 4. ตั้งค่าการเปลี่ยนเส้นทางสำหรับ ERC `eips.ethereum.org`เพื่อไปยังเว็บไซต์ใหม่
> 5. สร้างเอกสารรวมสำหรับให้บรรณาธิการกำหนดหมายเลข EIP/ERC เนื่องจากกระบวนการเหล่านี้มีความเชื่อมโยงกันอย่างลึกซึ้ง ผมจึงไม่คิดว่าเราจะมีความขัดแย้งกันของหมายเลขตรงนี้ ดังนั้น ผมจึงเสนอให้สร้างเอกสารที่มีหมายเลขถัดไปที่พร้อมใช้งาน เพื่อให้บรรณาธิการสามารถใช้หมายเลขนั้นในการกำหนดได้
> 
> นี่คือขั้นตอนขั้นต่ำสุดในการแยกกระบวนการออกจากกัน จากนั้นจะมีการเปลี่ยนแปลงเพิ่มเติมที่ต้องทำ เช่น ลบการอ้างอิงที่ไม่สมเหตุสมผล อัปเดตเทมเพลตเว็บไซต์ให้ไม่รวมหมวดหมู่ที่ไม่มีอยู่ ฯลฯ
> 
> ผมไม่อยากจมอยู่กับการถกเถียงไม่รู้จบเกี่ยวกับทุกขั้นตอนของการโยกย้าย หากการโยกย้ายได้รับการสนับสนุนตั้งแต่แรก เราก็ควรทำให้มันเกิดขึ้น แม้ว่าจะต้องใช้เวลาสักหน่อยในการขัดเกลา หากเรารอให้สมบูรณ์แบบ เราจะไม่สามารถเปลี่ยนแปลงอะไรได้เลย
> 
> สคริปต์

**Review by Cassieminkus1 — APPROVED** [review]

**Review by Danyalkasiri — COMMENTED** [review]
****

File: EIPS/eip-173.md (hunk 1 above)
Comment (Danyalkasiri) [reply EIPS/eip-173.md:5]: ********

Comment (Danyalkasiri) [reply EIPS/eip-173.md:5]: - [ ] ********

**Review by sins921 — APPROVED** [review]

**Review by Nithin200487 — APPROVED** [review]

**Review by sins921 — APPROVED** [review]

**Comment by Esmat330045** [comment]
approve the changes

**Review by tehran19r — APPROVED** [review]

**Comment by Sheriwaris** [comment]
https://github.com/ethereum/EIPs.git

File: EIPS/eip-173.md (hunk 1 above)
Comment (sanchezmaldonadojesusadrian14-coder) [reply EIPS/eip-173.md:5]: 🫡

**Comment by Mohammedsapry** [comment]
God

**Review by salasjuarezederaldair60-eng — COMMENTED** [review]
Ok

**Review by Themoor1 — APPROVED** [review]

File: EIPS/eip-173.md (hunk 1 above)
Comment (pepovintage-hub) [reply EIPS/eip-173.md:5]: Goodluck..

**Review by Gonzamany — APPROVED** [review]

**Comment by Estarossa97** [comment]
Ok

**Pull request merged**